core/transaction/transactionSorter.go

```diff
@@ -0,0 +1,44 @@
+package transaction
+
+import (
+	"bytes"
+	"sort"
+
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go-core/hashing"
+)
+
+// SortTransactionsBySenderAndNonceWithFrontRunningProtection - sorts the transactions by address and randomness source to protect from front running
+func SortTransactionsBySenderAndNonceWithFrontRunningProtection(transactions []data.TransactionHandler, hasher hashing.Hasher, randomness []byte) {
```

Comment:
please add also the legacy sorting, so we can re-index all data.

Reply:
fixed